workers/unity/Assets/Playground/Config/PlayerTemplate.cs

```diff
@@ -14,8 +14,16 @@ public static class PlayerTemplate
         {
             if (playerCreationArguments.HasValue)
             {
-                var deserializationSuccess = PlayerLifecycleHelper.DeserializeArguments<PlayerCreationParams>(
-                    playerCreationArguments.Value, out var deserializedArguments);
+                try
+                {
+                    var deserializedArguments = PlayerLifecycleHelper.DeserializeArguments<PlayerCreationParams>(
```

Comment:
Again - if we aren't using this, it doesn't belong in Playground. Rather unit or integration tests